hack/kind/setup.sh

```diff
@@ -61,11 +108,15 @@ run_registry() {
     http://local-registry:30000 || {
     die "Failed to reach local-registry running"
   }
-  echo -e "      ---------------------------------- \n"
+  line 50
 
 }
 
 setup_cluster() {
+  header "Setting up Cluster"
```

Comment:
can we add a check if cluster already exists use same cluster instead of failing. This will help if script was failed at some point and we need to re-run script

```
  🔆🔆🔆  Setting up Cluster  🔆🔆🔆
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
ERROR: failed to create cluster: node(s) already exist for a cluster with the name "obs-operator"
```